index.js

```diff
@@ -0,0 +1,120 @@
+const myLibrary = [];
+
+const formContainer = document.getElementById('form_container');
+const bookShelf = document.getElementById('book_shelf');
+
+let bookID = myLibrary.length - 1;
+function Book(title, pages) {
+  this.id = bookID;
+  bookID += 1;
+
+  this.title = title;
+  this.pages = pages;
+  this.read = false;
+}
+
+function deleteBook(bookContainer, book) {
+  bookContainer.parentNode.removeChild(bookContainer);
+  myLibrary.splice(book.id, 1);
+}
+
+function readBook(readButton, book) {
+  if (book.read === false) {
+    readButton.style.backgroundColor = 'green';
+  }
+  if (book.read === true) {
+    readButton.style.backgroundColor = 'red';
+  }
+  book.read = !book.read;
+}
+
+function addBook(book) {
+  const bookContainer = document.createElement('div');
+  bookContainer.setAttribute('class', 'bookContainer');
+  bookContainer.setAttribute('id', '');
+  bookShelf.appendChild(bookContainer);
+
+  const bookTitle = document.createElement('p');
+  bookTitle.textContent = book.title;
+
+  const bookPages = document.createElement('p');
+  bookPages.textContent = book.pages;
+
+  bookContainer.appendChild(bookTitle);
+  bookContainer.appendChild(bookPages);
+
+  const deleteButton = document.createElement('button');
+  deleteButton.addEventListener('click', () => { deleteBook(bookContainer, book); });
+  deleteButton.textContent = 'Delete';
+  bookContainer.appendChild(deleteButton);
+
+  const readButton = document.createElement('button');
+  book.read = true;
+  readBook(readButton, book);
+  readButton.addEventListener('click', () => { readBook(readButton, book); });
+  readButton.textContent = 'Read';
+  bookContainer.appendChild(readButton);
+}
+
+function addBookToLibrary() {
```

Comment:
- What's the difference between `addBook` and `addBookToLibrary` functions? Please just use the `addBookToLibrary` for adding a new book to keep your code organized and readable.

Reply:
addBook was used in the initial load of the myLibrary array, so it could load already existing elements. The initial load function has been removed since it is not a requirement, allowing the addBook logic to be moved into addBookToLibrary.